### PR TITLE
Fix linting error

### PR DESCRIPTION
**Pull request by zhangchiqing**
image

### CODE REVIEW

File: insecure/wintermute/attackOrchestrator_test.go [hunk 1]
```diff
@@ -146,8 +146,8 @@ func testConcurrentExecutionReceipts(t *testing.T,
 	receiptTargetIds, err := rootStateFixture.State.Final().Identities(filter.HasRole(flow.RoleAccess, flow.RoleConsensus, flow.RoleVerification))
 	require.NoError(t, err)
 
-	eventMap := make(map[flow.Identifier]*insecure.Event)
-	receipts := make([]*flow.ExecutionReceipt, 0)
+	var eventMap map[flow.Identifier]*insecure.Event
+	var receipts []*flow.ExecutionReceipt
```
Comment (zhangchiqing) [inline insecure/wintermute/attackOrchestrator_test.go:150]: The next line is initialing these two values, no need to init here.

File: ledger/complete/wal/fadvise.go [hunk 2]
```diff
@@ -20,6 +20,7 @@ func fadviseNoLinuxPageCache(fd uintptr, fsync bool) error {
 // If GOOS != "linux" then this function does nothing.
 // CAUTION: If fsync=true, this will call unix.Fsync which
 // can cause performance hit especially when used inside a loop.
+//nolint:unused,deadcode
```
Comment (zhangchiqing) [inline ledger/complete/wal/fadvise.go:23]: this code is only used by linux, macos never uses it, but the linter would complain unused or deadcode. 

skip this false negative

**Review by turbolent — APPROVED** [review]

**Review by fxamacker — APPROVED** [review]
LGTM! 👍  Thanks for fixing!

**Review by peterargue — APPROVED** [review]

**Review by ramtinms — APPROVED** [review]

**Comment by zhangchiqing** [comment]
bors merge

**Pull request merged**